apps/crn-server/scripts/export-analytics.ts

```diff
@@ -155,6 +169,7 @@ const transformRecords = (
       type,
```

Comment:
The payload's objectID should have the outputType included since objectIDs are unique